### PR TITLE
Program Changes Cag642

### PR DESCRIPTION
**Pull request by cag642**
Program changes given the comments from DJ.

### CODE REVIEW

**Review by bjones1 — CHANGES_REQUESTED** [review]
This pull request should be sent to the `main` branch, not the `dh24540` branch. Please review and resubmit.

**Comment by cag642** [comment]
I have changed the merge to bjones1 : main.

**Comment by cag642** [comment]
I have revised the files. I removed the comments left about formatting and fixed those. There are two comments I left, so that you can see what was recommended and why I did not change them. 

I am requesting a new review.

File: thorgame/dh2540/The Power of Thor.cpp [hunk 1]
```diff
@@ -1,28 +1,37 @@
-// # *The Power of Thor*
```
Comment (bjones1) [inline thorgame/dh2540/The Power of Thor.cpp:1]: Leave this file unchanged, since it's not your code.

File: PowerOfThor/cag642/Thor (1).cpp [hunk 2]
```diff
@@ -1,6 +1,11 @@
 // # Thor Program
```
Comment (bjones1) [inline PowerOfThor/cag642/Thor (1).cpp:2]: Rename this to `Thor.cpp` instead of `Thor (1).cpp`.

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by bjones1** [comment]
There are still edits to other's files -- chat with me to get this fixed.

**Comment by cag642** [comment]
I am so sorry, I just realized I never fixed this. I went into my fork and changed the DH file to what he has in main of the normal repo. For some reason that commit is not adding to this pull request though.

**Comment by bjones1** [comment]
I'd be happy to help -- when are you available?

**Comment by cag642** [comment]
I am available any time before 2:45

**Comment by cag642** [comment]
I am leaving for lunch, I will be back in 30 minutes.

**Comment by cag642** [comment]
Restarting with a new pull request

**Pull request closed without merging**